packages/xo-server/src/xo-mixins/rest-api.mjs

```diff
@@ -940,6 +940,29 @@ export default class RestApi {
         },
       },
     }
+    collections.groups.actions = {
+      update: withParams(
+        async ({ id, name, userIds }, req) => {
+          const group = await app.getGroup(id)
+          if (group.provider !== undefined) {
+            throw new Error('Cannot edit synchronized group')
+          }
+
+          if (name !== undefined) {
+            await app.updateGroup(id, { name })
+          }
```

Comment:
Avoid adding business logic into the REST API code.
consider the REST API as much as possible as an interface that exposes xo-servers methods via HTTP requests.
IMHO, you can move this logic to `xo-mixins/subjects.mjs#updateGroup`.

@fbeauchamp What do you think?